frontend/yarn.lock

```diff
@@ -2516,7 +2516,7 @@
   "resolved" "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz"
   "version" "2.0.6"
 
-"asn1.js@^5.2.0":
+"asn1.js@^5.0.0", "asn1.js@^5.2.0":
```

Comment:
nit: do these need to be checked in? same with package-lock
I've had to manually remove them from git locally, maybe .gitignore isn't picking up the stuff in frontend/

Reply:
yes, it looks like it should be checked in 
![image](https://user-images.githubusercontent.com/45133457/124374090-28cc4400-dc4d-11eb-8ea7-6e1e6bb7fc57.png)
https://classic.yarnpkg.com/en/docs/yarn-lock/
![image](https://user-images.githubusercontent.com/45133457/124374100-371a6000-dc4d-11eb-8509-e87cf191bba8.png)
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
ah I see, thanks for the info! I need to start doing this